backend/dataall/modules/shares_base/services/share_notification_service.py

```diff
@@ -109,6 +110,38 @@ def notify_persistent_email_reminder(self, email_id: str):
         )
         return notifications
 
+    def notify_managed_policy_limit_exceeded_action(self, email_id: str):
+        share_link_text = ''
+        if os.environ.get('frontend_domain_url'):
+            share_link_text = (
+                f'<br><br>Please visit data.all <a href="{os.environ.get("frontend_domain_url")}'
+                f'/console/shares/{self.share.shareUri}">share link</a> '
+                f'to view more details.'
+            )
+
+        msg_intro = f"""Dear User,
+        
```

Comment:
weird sentence: `We are contacting you because **for** a share requested by {email_id} failed` ---> We are contacting you because a share requested by {email_id} failed